test/framework/setup.go

```diff
@@ -642,17 +642,15 @@ func universalZoneProxyRelatedResource(
 
 		app, err := NewUniversalApp(
 			cluster.GetTesting(),
-			uniCluster.GetDockerBackend(),
 			uniCluster.name,
 			dpName,
 			"",
 			appType,
-			Config.IPV6,
-			false,
-			[]string{},
-			[]string{},
-			"",
-			concurrency,
+			UniversalAppRunOptions{
```

Comment:
Refactoring: introducing a new struct to wrap all these fields (because I wanted to add more)